_includes/graphql/getting-started.md

```diff
@@ -2,61 +2,55 @@
 
 [GraphQL](https://graphql.org/), developed by Facebook, is an open-source data query and manipulation language for APIs. In addition to the traditional [REST API](/rest/guide/), Parse Server automatically generates a GraphQL API based on your current application schema.
```

Comment:
This would be a good area to mention Relay support

"[GraphQL](https://graphql.org/), developed by Facebook, is an open-source data query and manipulation language for APIs. In addition to the traditional [REST API](/rest/guide/), Parse Server automatically generates a GraphQL API based on your current application schema. Specifically, Parse has opted for the [Relay](https://relay.dev/docs/en/introduction-to-relay) specification in-line with industry best-practices."

Reply:
We have a specific section dedicated to Relay, i think it's sufficient

Reply:
But let's add your suggestion (convert this to suggestion please) 😄

Reply:
Done